docs/release-process.rst

```diff
@@ -0,0 +1,41 @@
+
+Release process
```

Comment:
Primary caveat is the generic aspect of the heading/page title.
The contributing documentation already includes  release processes for components in other languages (Ansible, Java) e.g. see https://ome-contributing.readthedocs.io/en/latest/java-development.html#release-process

Would it be make sense to rebrandh this as the reference page for the development/release lifecycle of Python components under the OME organization?

Reply:
The process describes in this page is focusing on the versioning aspect of a repository not really the release process. `bump2version` is for example used in omero-insight see https://github.com/ome/omero-insight/blob/master/.bumpversion.cfg

Maybe `bump2version` could be also used out to the Java components.

What is missing from this page is a description on how a package is pushed to pypi.
this is clearly explained in https://github.com/ome/ome-contributing/pull/7/ for the Java components

Reply:
@jburel The "push to pypi" happens auto-magically, so I don't need to know how to do it.
If that needs to be included, can you describe what you want to see there?

Reply:
What is missing here:

Once the tag is validated, the master branch and the tag can also be pushed to the organization repository together:

* push the tag to the  organization repository. This will create and automatically upload the artifacts to pypi
* push the master branch

Reply:
This PR includes these instructions:

```
To complete the release, push the master branch and the release tag to origin::

    $ git push origin master v5.8.0
```

Reply:
I will add. This will automatically build the artifacts and push them to pypi

Reply:
I added ```Publishing to Pypi
==================

Many of the OME Python repositories use GitHub actions to pubish to Pypi
when a new Tag is created and pushed to GitHub.
This is typically specified in a file such as `.github/workflows/publish_pypi.yml`.```

Reply:
ok, that was probably added after my initial comment
That will do